core/src/main/java/org/kiwix/kiwixmobile/core/main/MainMenu.kt

```diff
@@ -164,6 +164,20 @@ class MainMenu(
     search.setOnMenuItemClickListener { navigateToSearch(zimFileReader) }
   }
 
+  fun hideBookSpecificMenuItems() {
+    search.isVisible = false
```

Comment:
the `setVisibility` method was made for this

Reply:
Completely missed that one. Cool!